hack/remove-members.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var dryrun bool
+var repoRoot string
+
+func main() {
+	flag.StringVar(&repoRoot, "root", ".", "Root of the repository")
+	flag.BoolVar(&dryrun, "dryrun", true, "Enable Dryrun or not")
+
+	flag.Parse()
+
+	if len(flag.Args()) < 1 {
+		fmt.Print("Usage: remove-members [flags] <file-containing-members-list>\n\nFlags:\n\t-root\tstring\tpath to root directory of the repository\n\t-dryrun\tboolean\tEnable/Disable dryrun (Default: Enabled)")
+		os.Exit(0)
+	}
+
+	memberList, err := readMemberList(flag.Args()[0])
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	configPath := repoRoot + "/config"
+
+	err = removeMembers(memberList, configPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+}
+
+//readMemberList reads the list of members to be removed from the given filepath
+func readMemberList(path string) ([]string, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	var members []string
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		members = append(members, scanner.Text())
+	}
+	return members, scanner.Err()
+}
+
+//removeMembers walks through the config directory and removes the occurences of the given member name
+func removeMembers(memberList []string, configPath string) error {
+	for _, member := range memberList {
+		var orgs, teams []string
+		count := 0
+		fmt.Print(member)
```

Comment:
would be better to `log` here and a more descriptive log message

Reply:
Should I change all instances of `fmt` to `log`?

Currently the output looks like 
```
~/P/o/hack (remove-members-tool|…) ❯❯❯  go run remove-members.go -path=../config/kubernetes members.txt
nikhita
 Orgs: [kubernetes]
 Teams: [sig-contributor-experience sig-release]
 Number of occurences: 3

Commit Command: "commit -m Remove nikhita from the kubernetes org -m Remove nikhita from sig-contributor-experience, sig-release teams"

daemon1024
 Orgs: []
 Teams: []
 Number of occurences: 0
```

P.s. I don't really want to remove you :P just for showing the output :)

Reply:
I think fmt is also ok, but would be good to use a more descriptive message :)